packages/api/src/Api.ts

```diff
@@ -46,11 +63,7 @@ export class Api extends ApiPromise {
      */
     cennzxSpot?: CennzxSpot;
 
-    constructor(provider: ApiOptions | ProviderInterface = {}) {
-        const options =
-            isObject(provider) && isFunction((provider as ProviderInterface).send)
-                ? ({provider} as ApiOptions)
-                : ({...provider} as ApiOptions);
+    constructor(options: ApiOptions) {
```

Comment:
we still need these transition codes in constructor

Reply:
sure